src/test/java/racing/domain/RacingCarsTest.java

```diff
@@ -2,32 +2,25 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.List;
 
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 class RacingCarsTest {
 
-
     @DisplayName("우승자를 찾는다.")
     @Test
     void findWinners() {
-        final List<String> namesOfCars = getDefaultNamesOfCars();
-        final RacingCars racingCars = new RacingCars(namesOfCars);
-
-        racingCars.run(getDefaultRule());
-
-        assertThat(racingCars.findWinners()).hasSize(namesOfCars.size());
-    }
-
-    GameRule getDefaultRule() {
-        return () -> true;
-    }
-
-    List<String> getDefaultNamesOfCars() {
-        return Arrays.asList("solar", "holar", "cola");
-
+        final List<Car> cars = new ArrayList<>();
+        cars.add(new Car("solar", new Location(3)));
+        cars.add(new Car("kuku", new Location(4)));
+        cars.add(new Car("cola", new Location(1)));
+        final RacingCars racingCars = RacingCars.of(cars);
+
+        final List<Car> winners = racingCars.findWinners();
+        assertThat(winners).hasSize(1);
+        assertThat(winners).contains(new Car("kuku", new Location(4)));
```

Comment:
```suggestion
        assertThat(winners).containsExactly(new Car("kuku", new Location(4)));
```

containsExactly를 사용하면 사이즈에 대한 검증도 하나의 assertThat으로도 가능할 것 같아요 :)
순서가 상관 없다면 containsExactlyInAnyOrder() 메소드를 사용할 수도 있으니 참고부탁드려요~